### PR TITLE
Remove quota feedback if no link set

### LINKED ISSUE #11635: Incorrect behavior of "{space} used" tab
14.0.2 RC 1

### Steps to reproduce
1. Go to "Delete files" tab
2. Go to "{space} used" tab

### Expected behaviour
Something

### Actual behaviour
Nothing

### Screenshots:
image

### PR DESCRIPTION
**Pull request by skjnldsv**
Fix #11635

### CODE REVIEW

File: apps/files/css/files.scss [hunk 1]
```diff
@@ -770,6 +770,14 @@ table.dragshadow td.size {
 	border-radius: 0;
 	background-color: transparent;
 	z-index:1;
+	
+	> a[href='#'] {
+		// if no ink is set, no mouse feedback
```
Comment (ChristophWurst) [inline apps/files/css/files.scss]:
```suggestion
		// if no link is set, no mouse feedback
```

**Review by ChristophWurst — CHANGES_REQUESTED** [review]
Even small diffs can have typos

**Review by ChristophWurst — APPROVED** [review]

**Review by MorrisJobke — APPROVED** [review]
Tested and works 👍

**Comment by jancborchardt** [comment]
Good stuff! :) I’m just wondering if removing the interface feedback of the link will not still make it confusing for screenreader users as it’s an empty link? I would have thought it should be removed in the PHP/JS already. But cc @nextcloud/accessibility for input. :)

**Pull request merged**